guide/C/ch_invest.xml

```diff
@@ -1817,6 +1817,1129 @@ Income
+         (<menuchoice>
+            <guimenu>Reports</guimenu>
+            <guisubmenu>Income &amp; Expense</guisubmenu>
+            <guimenuitem>Trial Balance</guimenuitem>
+          </menuchoice>) bottom line total debits will not balance to 
+          total credits.
+      </para>
+                    
+      <sect3 id="invest-sellLimitations">
+        <title>Limitations</title>
+        
+        <para>There are some limitations to using this feature.</para>
+        
+        <itemizedlist>
+          <listitem>
+            <para><application>&app;</application> has only 1 method of
+              automatically linking buy transactions to sell transactions -
+              First In/ First Out (FIFO). I.e. The oldest securities are always
+              sold first.
+            </para>
+          </listitem>
+          
+          <listitem>
+            <para>The Advanced Portfolio Report does not use lot information
+              when calculating costs. It calculates the cost basis and gains
+              or losses based on the selected <emphasis>Basis calculation method
+              </emphasis> report option (Average, FIFO or LIFO). I.e. it does
+              not use any of the lot information, just the security transaction
+              splits.
+            </para>
+          
+            <para>If you are selling <emphasis>all</emphasis> of your holding of
+              a particular security, it doesn't matter which cost method you
+              use as the total cost of the securities sold will be the same.
+              It is only when you have partially sold a security, that the 
+              costing method is important, so that the Advanced Portfolio Report
+              costs (and therefore gains or losses) are correct for the 
+              remainder of the security holding going forward. As the costing
+              method is a report option, it is preferable that the same costing
+              method is used for all securities. Otherwise, it may be necessary
+              to run multiple Advanced Portfolio Reports, 1 for each costing
+              method and only including the securites using that costing method.
+            </para>
+            
+            <note>
+              <para>Provision has been made in <application>&app;</application>
+                for specifying costing method per security account but this is
+                yet to be implemented.
+              </para>
+            </note>
+        
+            <para>Given that the Advanced Portfolio Report doesn't use any of
+              the lot information, if you want the reported costs and
+              gains/losses to be consistent with the transaction/lot data, it is
+              pointless (other than for the automatic creation of the gain/loss
+              transaction) linking sales with specific acquisitions unless you
+              always sell a particular security LIFO (Last In, First Out). I.e.:
+            </para>
```

Comment:
This doesn't make sense to me. "If you want the reported costs and gains/losses to be consistent with the transaction/lot data, it is pointless...linking sales with specific acquisitions unless you always sell.. LIFO". 

If you want the APR and your books to be consistent and gains in your books are calculated FIFO with the Lots function, shouldn't the APR also be told to use FIFO?